user_management/ui/tests/test_views.py

```diff
@@ -76,3 +76,21 @@ def test_get_registered_user(self):
             self.view_class.already_verified_message,
             str(request._messages.store[0]),
         )
+
+    query_string = 'validated'
+
+    @override_settings(VALIDATED_QUERYSTRING=query_string)
+    def test_get_redirect_url_with_query(self):
+        view = views.VerifyUserEmailView
+        self.already_verified = False
```

Comment:
If you do the `as_view` method then you will just need to do `view.already_verified = False`